### ISSUE TITLE
Implement explicit Variation Set

### ISSUE THREAD

**Issue by reece** [ga4gh/vr-spec#116]
VariationSets are essential to ClinGen's goal to represent ClinVar variation.

This issue is to collect input on and implement a static Variation Set defined by explicit enumeration of constituent variation.

**Comment by reece** [comment]
The current direction is this hierarchy:
```
Variation
+- DiscreteVariation
   +- Allele
   +- (Translocation?)
+- AggregateVariation
   +- DiscreteVariationSet
   +- Haplotype (of DiscreteVariation, with constraints)
   +- Genotype (of Haplotypes)
   +- VariationSet (unconstrained set of Variation)
```

**Comment by reece** [comment]
Status: merged master into 116-variation-sets to pick up the changes from #173 . Now working on getting the enref/deref code to work for variation sets. No problems expected.

**Comment by reece** [comment]
Implemented enref/deref in Implemented enref/deref in 152fae1. Need to add tests to vr-python and vr-spec.
